### DISCUSSION TITLE
Achievement `Heart On Your Sleeve`

### DISCUSSION THREAD

**Discussion by georgetoloraia** [community/community#102491] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

## Hello to all..!

- Can you tell me how i can get Achievement  `Heart On Your Sleeve`:

**Comment by surendratiwari3** [comment, marked as answer]
the 'Heart On Your Sleeve' & 'Open Sourcerer' achievement badges criteria are not yet released to  users at the moment.

**Reply by georgetoloraia** [reply]
ok, thank for answer

**Reply by cloudglides** [reply]
❤️

**Reply by RafaDEV99** [reply]
❤️

**Reply by Toriki1111** [reply]
❤️

**Reply by Jimmi1e** [reply]
❤️

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @georgetoloraia. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
the 'Heart On Your Sleeve' & 'Open Sourcerer' achievement badges criteria are not yet released to users at the moment.

**Comment by codewithpapakwame** [comment]
❤️

**Comment by codewithpapakwame** [comment]
To get heart on your sleeve mostly React to something on Github with a ❤️ emoji (Being tested)

**Reply by codewithpapakwame** [reply]
❤️ 2 times to get the badge

**Reply by ewudes** [reply]
❤️

**Comment by Im-Siddhant** [comment]
react

**Reply by ewudes** [reply]
❤️

**Comment by Maxim17081983** [comment]
Я хочу

041B232BF35A8002005109976630451469D3A048344C7CAD


Вт, 27 янв. 2026 г. в 02:07, ewudes ***@***.***>:
